### PR TITLE
[docs] Testing guide polish

### PR DESCRIPTION
**Pull request by drarmstr**
Minor polish for new guide on testing from @timbergus #1198

### CODE REVIEW

File: docs/docs/guides/testing.md [hunk 1]
```diff
@@ -2,27 +2,27 @@
 title: Testing
 ---
 
-## Testing Recoil Selectors inside of a React Component
+## Testing Recoil state inside of a React component
```
Comment (timbergus) [inline docs/docs/guides/testing.md:5]: Just for consistency, `state` in the second block is capitalized (`State`). Shouldn't be this one capitalized too? Or the other one in lower case.

File: docs/docs/guides/testing.md [hunk 2]
```diff
@@ -49,26 +49,26 @@ export default Form;
 #### Test
 
 ```jsx
-import { RecoilRoot } from 'recoil';
-import { fireEvent, render, screen } from '@testing-library/react';
+import {RecoilRoot} from 'recoil';
```
Comment (timbergus) [inline docs/docs/guides/testing.md:52]: Just out of curiosity. Why not adding spaces around objects? I knew the rule about not leaving spaces around array contents, but I thought leaving spaces between object contents and the curly braces was acceptable.

**Review by timbergus — COMMENTED** [review]
Thanks a lot for the review! It has more sense in talking about `nodes` than `elements`. And thanks for rephrasing the first paragraph. English is not my mother tongue, and sometimes I write in English thinking in Spanish 😅

File: docs/docs/guides/testing.md (hunk 1 above)
Comment (drarmstr) [reply docs/docs/guides/testing.md:5]: thanks for the catch!

File: docs/docs/guides/testing.md (hunk 2 above)
Comment (drarmstr) [reply docs/docs/guides/testing.md:52]: Yeah, spaces there are pretty common with open source projects.  However, the internal Facebook prettier standard has no spaces.  So, this change is just for consistency with the rest of the code base and samples.  No value judgement either way.

Comment (timbergus) [reply docs/docs/guides/testing.md:52]: Thanks. I didn't know 🙂

**Pull request merged**